tests/unit/test-util/test.artifacts.js

```diff
@@ -55,4 +55,74 @@ describe('prepareArtifactsDir', () => {
     }
   ));
 
+  it('throws an UsageError when it lacks permissions to stat the directory',
+    () => withTempDir(
+      function(tmpDir) {
```

Comment:
Hmm, I don't think this is quite right. I believe that in order to use `this.skip()` you'll need to make two changes:
* Pass a `function` to `it()` so that mocha will bind `this` to the test case instance
* Use arrow functions everywhere else in the test case so that `this` won't be bound to an instance

Example:
````js
it('throws an UsageError when it lacks permissions to stat the directory',
  function() {
    return withTempDir(
      (tmpDir) => {
        if (process.platform === 'win32') {
          // Now `this` will be bound to the mocha test case instance:
          this.skip();
          return;
        }
      }
    );
  }
);
````

More info: https://mochajs.org/#inclusive-tests